### PR TITLE
 izel made major changes. We can now acsess tasks name and user names…

### PR DESCRIPTION
**Pull request by Saidac**
Izel made major changes. We can now access task name and user names in show view. Theres also a link from the headers in index view to show that works.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -220,22 +142,32 @@ app.get('/:uuid', (req, res) => {
...
+          name: challenge.name,
+          tasks: tasks.map((task) => {
+            var owner = users.filter((user) => {
+              return user.id === task.UserId;
+            })[0];
+
+            return Object.assign({}, task.dataValues, { owner: owner.dataValues });
+          })
+        };
+        console.log(dataStructure.tasks[1]);
```
Comment (izelnakri) [inline index.js]: remove console logs

File: index.js [hunk 2]
```diff
@@ -54,141 +54,59 @@ app.get('/wait', (req, res) => {
...
-      var mailOptions = {
-        to: req.body.participant.email,
-        subject: 'Invitation to challenge',
-        text: ` Hello,
-          You has been invited to do a challenge, Please click this below link to see the details
-          http://localhost:3000/${savedChallenge.uuid}`
-      };
+    });
+  }).then((participantTask) => {
+    console.log('RESULT IS:');
```
Comment (izelnakri) [inline index.js]: remove console.log

File: views/chainz/index.pug [hunk 3]
```diff
@@ -8,10 +8,10 @@ block content
         table.cz-challenge-table
           thead
             tr
-              td(colspan="7"): h5 Challenge name
+              td(colspan="7"): h5=challenge.name
           tbody
             tr
-              td: h5 User
+              td: h5=user.user
```
Comment (izelnakri) [inline views/chainz/index.pug]: what is user.user?

Comment (Saidac) [reply views/chainz/index.pug]: This resource is a residue from sketching the show view. Will delete it.

**Pull request merged**